src/test/java/com/google/javascript/clutz/partial/static_method.js

```diff
@@ -0,0 +1,16 @@
+goog.provide('foo.bar.Class');
+
```

Comment:
Can you add docs to this about what it's testing, what the expected output should include?

Reply:
Added - do they answer your question adequately?